<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>478</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>478</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Environment profile generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="generalGroupBox">
      <property name="title">
       <string>General info</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="portNumberLabel">
         <property name="text">
          <string>Port number:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="launchCommandLabel">
         <property name="text">
          <string>Launch command:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="portNumberLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>8888</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="launchCommandLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>robots/Environment</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="objectGroupBox">
      <property name="title">
       <string>Object details</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="objectIdLabel">
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="5">
        <widget class="QLineEdit" name="objectIdLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="startPositionLabel">
         <property name="text">
          <string>Start position:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="generateCheckBox">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>Random</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="startPositionXLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>2000</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="startPositionYLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>2000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="sizeLabel">
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="5">
        <widget class="QLineEdit" name="sizeLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>300</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="intersectionLabel">
         <property name="text">
          <string>Intersection type: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="5">
        <widget class="QComboBox" name="intersectionComboBox">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <item>
          <property name="text">
           <string>Allowed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Denied</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Allowed for same color</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="movableLabel">
         <property name="text">
          <string>Movable:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QRadioButton" name="movableYesRadioButton">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>yes</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QRadioButton" name="movableNoRadioButton">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>no</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="orientationLabel">
         <property name="text">
          <string>Orientation:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="5">
        <widget class="QLineEdit" name="orientationLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>45</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="velocityLabel">
         <property name="text">
          <string>Velocity:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="5">
        <widget class="QLineEdit" name="velocityLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="colorLabel">
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1" colspan="5">
        <widget class="QLineEdit" name="colorLineEdit">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="text">
          <string>#ff00ff</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3" colspan="3">
        <widget class="QPushButton" name="addObjectPushButton">
         <property name="text">
          <string>Add object</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>,</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>234</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="clearAllPushButton">
      <property name="text">
       <string>Clear all</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="generateProfilePushButton">
      <property name="text">
       <string>Generate profile</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>addObjectPushButton</tabstop>
  <tabstop>generateProfilePushButton</tabstop>
  <tabstop>clearAllPushButton</tabstop>
  <tabstop>startPositionXLineEdit</tabstop>
  <tabstop>startPositionYLineEdit</tabstop>
  <tabstop>sizeLineEdit</tabstop>
  <tabstop>intersectionComboBox</tabstop>
  <tabstop>movableYesRadioButton</tabstop>
  <tabstop>movableNoRadioButton</tabstop>
  <tabstop>orientationLineEdit</tabstop>
  <tabstop>velocityLineEdit</tabstop>
  <tabstop>colorLineEdit</tabstop>
  <tabstop>portNumberLineEdit</tabstop>
  <tabstop>launchCommandLineEdit</tabstop>
  <tabstop>objectIdLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
